demo/index.html

```diff
@@ -23,6 +23,7 @@
       href="https://cdn.jsdelivr.net/npm/prismjs@1.24.1/themes/prism.css"
     />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
```

Comment:
Please update this area to be like the following code example

https://github.com/AlaskaAirlines/auro-avatar/blob/2a95c410cb8e02f91c1c4868305792fc4c1754b1/demo/index.html#L12-L17

Reply:
Ok, sure i will updated. Thanks